lib/game.rb

```diff
@@ -13,8 +14,8 @@ def initialize(display, presenter, board, players)
     @player_1 = players[0]
     @player_2 = players[1]
 
+    @game_checker = GameChecker.new
     @active_player = @player_1
```

Comment:
Why not pass GameChecker to board?  Board can ask GameChecker about the result.

Reply:
Ooo good point!